crates/bitwarden-vault/src/cipher/cipher.rs

```diff
@@ -389,6 +408,14 @@ fn build_subtitle_identity(first_name: Option<String>, last_name: Option<String>
     sub_title
 }
 
+fn build_subtitle_ssh_key(fingerprint: Option<String>) -> String {
+    if let Some(fingerprint) = &fingerprint {
+        return fingerprint.clone();
+    }
+
+    "".to_string()
+}
+
```

Comment:
**suggestion**: We can add a `unwrap_or_default` to line 348 and get rid of this function.